modules/stringtie/stringtie/main.nf

```diff
@@ -48,6 +48,7 @@ process STRINGTIE {
         -A ${prefix}.gene.abundance.txt \\
         -C ${prefix}.coverage.gtf \\
         -b ${prefix}.ballgown \\
+        -p $task.cpus \\
         $options.args
 
     stringtie --version > ${software}.version.txt
```

Comment:
Fancy bumping the version too whilst you are here?
https://anaconda.org/bioconda/stringtie

Reply:
Need to change the version command to use `echo` too.

Reply:
Sure! Did not realised :sweat: